testsuite/src/it/java/io/quarkus/ts/startstop/utils/Logs.java

```diff
@@ -71,6 +71,9 @@
     private static final Pattern stoppedPatternControlSymbols = Pattern.compile(".* stopped in .*188m([0-9\\.]+).*", Pattern.DOTALL);
 
     private static final Pattern warnErrorDetectionPattern = Pattern.compile("(?i:.*(ERROR|WARN).*)");
```

Comment:
What about just this:
```suggestion
    private static final Pattern warnErrorDetectionPattern = Pattern.compile("(?i:.*(ERROR|WARN|SLF4J).*)");
```

Reply:
I thought about it, but end up with the approach of `blackListedLinesPattern[]` just in case if in the future there are some more unwanted lines appears.
For now, your approach looks sharp, should I change it?

Reply:
yea, I would go with this approach for now